.github/workflows/windows.yml

```
@@ -32,27 +32,40 @@ jobs:
           pathInArchive: "binaryen-/bin/wasm-opt.exe"
 
       - name: Checkout sources & submodules
-        uses: actions/checkout@master
+        uses: actions/checkout@v2.4.0
         with:
           fetch-depth: 1
           submodules: recursive
 
       - name: Install toolchain
         id: toolchain
-        uses: actions-rs/toolchain@master
+        uses: actions-rs/toolchain@v1.0.7
```

```suggestion
        uses: actions-rs/toolchain@v1
```
It is recommended to use this tag as it following updates to the latest minor version